src/server/actions/beef/arbiterChangeStatus.ts

```diff
@@ -0,0 +1,31 @@
+"use server"
+
+import { eq } from "drizzle-orm"
+import { and } from "drizzle-orm/sql/expressions/conditions"
+import { Address } from "viem"
+import { getArbiterStatuses } from "@/server/actions/lib/beef"
+import { Beefs, db } from "@/server/db/db"
+import { activeChain } from "@/utils/chain"
+
+export const refreshArbiterStatusesAction = async (beefAddress: Address) => {
+  const beef = await db.query.Beefs.findFirst({
+    where: (Beefs, { eq, and }) => and(eq(Beefs.chainId, activeChain.id), eq(Beefs.address, beefAddress)),
+    columns: { arbiters: true },
+  }).execute()
+
+  if (beef === undefined) {
+    throw new Error(`Beef with address ${beefAddress} does not exist`)
```

Comment:
Throw or return `true` seems like a bit of a weird pattern to me. Shouldn't we return a unified type? `bool`, `Ok | Error<string>` or just `void`?

Reply:
Agreed, I put the `true` as a placeholder. `void` makes way more sense for now